fractions/javaee/jaxrs-cdi/README.adoc

```diff
@@ -0,0 +1,6 @@
+# JAX-RS + CDI
+
+An internal fraction providing integration between JAX-RS
+and CDI.
+
+Please see the documentation for the JAX-RS and CDI fractions.
```

Comment:
The convention across the docs organisation is to write these sentences like this:

_For more information, see the JAX-RS or CDI fraction documentation._

Here, _JAX-RS_ and _CDI_ are links to the docs in question, or:

_For more information, see the CHAPTER_NAME chapter of the JAX-RS documentation, or the CHAPTER_NAME chapter of the CDI fraction documentation._ 

Here, _CHAPTER_NAME_ are links to the specific chapters of the respective documentations.

You can also write the sentences without linking to the documentations, but it facilitates the reader's job a great deal to be able to just click a link; aside from that, it is considered a good practice.

Reply:
That makes sense.

Reply:
If acceptable, I'll avoid links for now, as these READMEs are 

* Useful (maybe?) from with GitHub, which has different structure than the ref-guide where they also get slurped, and have (at the moment) different link/directory structures.
* The ref-guide is being restructured (Real Soon Now) and no links seems better than possibly broken links.